dist/handsontable.js

```diff
@@ -35137,8 +35137,14 @@ var _default = function _default() {
     maxCols: Infinity,
```

Comment:
I have the same request as for the `dist/handsontable.full.js` file.

Reply:
@swistach - my bad, I read the contributing guidelines and build docs but forgot ;) Reverted now.